### DISCUSSION TITLE
Switch A and B Button in frontend menu

### DISCUSSION THREAD

**Discussion by shaaapest** [rg35xx-cfw/rg35xx-cfw.github.io#239] [Q&A]
I want to use A to confirm (just like a GameBoy) and B for going back in the front menu. Is that anyhow possible? It´s super annoying to navigate the menu differently than the game you are starting afterwards.

**Comment by rebeltaz** [comment]
> I want to use A to confirm (just like a GameBoy) and B for going back in the front menu. Is that anyhow possible? It´s super annoying to navigate the menu differently than the game you are starting afterwards.

You can do this in Settings:
Main Menu > System Settings > Frontend Developer Options > Switch Confirm & Cancel Buttons in EmulationStation

**Reply by shaaapest** [reply]
Thank you so much 🙏🏼
